src/components/calendar/EventModal.js

```diff
@@ -0,0 +1,187 @@
+import React, {useState} from 'react';
+import {Button, Modal, Image, Card} from 'react-bootstrap';
+import './EventModal.css';
+
+function checkIfUndefined(attribute) {
+
+    if(typeof(attribute) !== "undefined") {
+        return attribute;
+    }
+
+    return '';
+}
+
+function formatAMPM(date) {
+    let hours = date.getHours();
+    let minutes = date.getMinutes();
+    let ampm = hours >= 12 ? 'pm' : 'am';
+    hours = hours % 12;
+    hours = hours ? hours : 12; // the hour '0' should be '12'
+    minutes = minutes < 10 ? '0'+minutes : minutes;
+    let strTime = hours + ':' + minutes + '' + ampm;
+    return strTime;
+}
+
+function getWhen(props) {
+        const months = [ "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December" ];
```

Comment:
I'm not too sure yet, but instead of re-writing the months in multiple jsx files, you can pass them through the props. I believe Calendar.js is related to EventModal, so you can use the same array from Calendar.js in EventModal to save from the repetition.